_widget/src/components/article/component.vue

```diff
@@ -41,24 +41,23 @@ export default {
   },
   watch: {
     'article' () {
-      this.prepare();
+      return this.prepare();
     }
   },
   mounted () {
-    this.prepare();
+    return this.prepare();
   },
   methods: {
     fixLinks () {
       [...this.$el.querySelectorAll('a')].forEach((a) => {
```

Comment:
Here, we set all URLs to be absolute. For links that should open in a new tab, `app.visit` doesn't touch them (eg. no need for `window.location.href = ...` as it just functions as a normal link).